allensdk/core/dataframe_utils.py

```diff
@@ -158,3 +158,48 @@ def enforce_df_int_typing(input_df: pd.DataFrame,
                 input_df[col] = \
                     input_df[col].fillna(INT_NULL).astype(int)
     return input_df
+
+
+def return_one_session_metadata(input_table: pd.DataFrame,
```

Comment:
this should really be more generic than it is since this file contains generic dataframe functions and this function is an outlier

Reply:
I don't really agree. While this could be more generically used, the error messages are meant to convey more descriptive information to a user trying to pull a session using `get_whatever_session` rather than a generic KeyError or similar.

Reply:
You would just have to rename the function to something more generic and rename `session_id` to something like `index_value` and rename the word `session`. It would still output a descriptive message like you have just without the word "session"

Reply:
done